CollAction/.gitignore

```diff
@@ -1,3 +1,5 @@
 wwwroot/js/bundle.js
 node_modules
-log*
\ No newline at end of file
+log*
+
+Views/Projects/Participate.cshtml
```

Comment:
Why exclude this?

Reply:
It is an incomplete file that I accidentally committed and pushed earlier.

Reply:
Could you undo it? Because it'll be merged to the friesland branch as it is now?